.github/pull_request_template.md

```diff
@@ -0,0 +1,7 @@
+#### Description of Change
+
+#### Things to Do Before Submitting
+
+* Have you signed the [Sonatype CLA](https://sonatypecla.herokuapp.com/sign-cla)?
+* Have you added and run automated tests for your change? (See the README.md.)
```

Comment:
Trivial: Feel like README could be a link if you're feeling nice.  Also, don't know the grammar rules but the period inside the parenthesis looks weird.  If a distinct sentence, why even have the parenthesis?

Reply:
done.